tests/convert_valid_utf32_to_latin1_tests.cpp

```diff
@@ -0,0 +1,40 @@
+#include "simdutf.h"
+
+#include <array>
+#include <iostream>
+
+#include <tests/helpers/transcode_test_base.h>
+#include <tests/helpers/random_int.h>
+#include <tests/helpers/test.h>
+#include <memory>
+
+namespace {
+  std::array<size_t, 7> input_size{7, 16, 12, 64, 67, 128, 256};
```

Comment:
You define this variable (`input_size`) here. Later, you redefine it as `std::array<size_t, 4> input_size{7,16,24,67};`. That's probably not what you want to be doing?